pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -127,6 +129,7 @@
   private final String _helixZkURL;
   private final String _helixClusterName;
   private final String _instanceId;
+  private final String _controllerParticipantInstanceId;
```

Comment:
I'll remove this field and directly use _instanceId. In the constructor, directly pass in `Controller_xxx`